packages/ui/src/components/TextArea/index.tsx

```diff
@@ -187,12 +191,11 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
                   variant="bodyStrong"
                   sentiment="neutral"
                   htmlFor={id ?? localId}
+                  prominence="strong"
```

Comment:
There is small issue with the disabled state, not related to your changes tho
<img width="1046" alt="Screenshot 2024-09-17 at 14 09 37" src="https://github.com/user-attachments/assets/c08b59ea-db14-46e3-b267-3b61696eb825">